js-jasmine/spec/agedBrie.spec.js

```diff
@@ -0,0 +1,29 @@
+const AgedBrie = require("../src/agedBrie");
```

Comment:
I like how you've split all the 'special' items and conditions into different specs and classes